.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "tools/landusedata"]
+	path = tools/landusedata
+	url = https://github.com/NGEET/tools-fates-landusedata
```

Comment:
I suggest adding the following which would only be used by git-fleximod for CESM:

``` yaml
            # Two settings only used when git-fleximod is used (for CESM)
            # Always require this submodule since it's small so doesn't hurt to have it
            fxrequired = AlwaysRequired
            # Standard Fork to compare to with "git fleximod test" to ensure personal forks aren't committed
            fxDONOTUSEurl = https://github.com/NGEET/tools-fates-landusedata
```

Reply:
Thanks.  I also added the `fxtag` since the landuse tool has tags.  Testing this out with `git-fleximod` resolves the issue I was seeing above.